src/main/java/org/fundacionjala/coding/kevinherrera/stringsorter/StringSorter.java

```diff
@@ -0,0 +1,55 @@
+package org.fundacionjala.coding.kevinherrera.stringsorter;
+
+import java.util.Arrays;
+
+/**
+ * @author KevinHerrera - AWT-[01].
+ * @version 0.1
+ */
+public class StringSorter {
+
+    public static final int INT_4 = 4;
+
+    /**
+     * sort inner letter of each word on a string.
+     * @param stringToSort string to sort
+     * @return string with inner letter of words sorted
+     */
+    public String sortInner(final String stringToSort) {
+        StringBuilder builder = new StringBuilder();
+        if (stringToSort == null) {
+            return builder.toString();
+        }
+        String[] strings = stringToSort.split(" ");
+        for (String c : strings) {
+            if (c.length() < INT_4) {
+                builder.append(c);
+                builder.append(" ");
+                continue;
+            }
+            char[] charArray = c.toCharArray();
+            Arrays.sort(charArray, 1, charArray.length - 1);
+            builder.append(inverseInnerAppend(charArray));
+            builder.append(" ");
+        }
+        if (builder.charAt(builder.length() - 1) == ' ') {
+            builder.deleteCharAt(builder.length() - 1);
+        }
+        return builder.toString();

```

Comment:
This may be refactored like this
```java
    String sortInner(final String stringToSort) {
        StringJoiner builder = new StringJoiner(" ");
        if (stringToSort == null) {
            return builder.toString();
        }
        String[] words = stringToSort.split(" ");
        for (String word : words) {
            if (word.length() < INT_4) {
                builder.add(word);
            } else {
                String[] charArray = word.split("");
                Arrays.sort(charArray, 1, charArray.length - 1, Collections.reverseOrder());
                builder.add(String.join("", charArray));
            }
        }
        return builder.toString();
    }
```